### ISSUE TITLE
Label sovrapposta al valore degli input readonly o disabled pre-popolati

### ISSUE THREAD

**Issue by ilbassa** [italia/bootstrap-italia#347]
## Comportamento atteso
La label dei form _readonly_ o _disabled_ già valorizzati dovrebbe spostarsi sopra l'input

## Comportamento attuale
Attualmente il valore pre-popolato viene visualizzato sovrascritto alla label

## Possibili soluzioni
Se il value è valorizzato bisognerebbe impostare la classe active nella label. È possibile pensare di farlo via javascript su tutti i campi del form?

**Comment by ilbassa** [comment]
Il problema si verifica solamente quando si carica una parte di form da una chiamata Ajax.
Il codice JS che "sposta" le label è presente alla riga 101 del file `js/forms.js` ed è lanciato, presumo, al caricamento della pagina.

La soluzione sarebbe quella di lanciare la stessa funzione sull'evento `$(document).ajaxComplete();` o similare.

Nei nostri applicativi aggiungeremo una funzione custom per gestire questo caso, pensate si possa introdurre come fix nel pacchetto js di boostrap-italia?
